README.md

```diff
@@ -67,6 +67,18 @@ ONEAUTH_CLIENT_ID=<client_id>
 ONEAUTH_CLIENT_SECRET=<client_scret>
 ```
 
+## Create symlinks
```

Comment:
## Fixing Windows Symlink Issue

If you are a windows users and facing error while starting the development server try running the following command:
```bash
bash ./bin/windows_setup.sh
```
*You might be needed to enable Windows Subsystem for Linux feature on your OS or run the command inside git bash.*